dependency-check-maven-config.xml

```diff
@@ -84,4 +128,103 @@
    ]]></notes>
         <cve>CVE-2014-0003</cve>
     </suppress>
+
+    <suppress>
+        <notes><![CDATA[
+        this CVE is generating a lot of false positives it should only include jackson-dataformat-xml jar
+         but it is catching all jackson dependencies unfortunately I can't fix it so if we start depending
+          on a vulnerable version of jackson-dataformat-xml this will sill suppress it.
```

Comment:
Could you maybe specify the file hashes of the jars in question to mitigate the risk that we might someday add a vulnerable `jackson-dataformat-xml` dependency?